R/plotly_build.R

```diff
@@ -561,7 +561,7 @@ train_data <- function(data, trace) {
     idx2 <- seq.int(2, NROW(dat), by = 2)
     dat[idx2, "x"] <- data[["xend"]]
     dat[idx2, "y"] <- data[["yend"]]
-    data <- dplyr::group_by_(dat, ".plotlyGroupIndex", add = TRUE)
+    data <- dplyr::group_by(dat, !!rlang::sym(".plotlyGroupIndex"), .add = TRUE)
```

Comment:
I don't think this would work for dplyr < 1.0, right? It might be useful to have a helper like:

```r
group_by_add <- function(..., add = TRUE) {
  if (packageVersion('dplyr') >= '1.0') {
      dplyr::group_by(...,  .add = add)
  } else {
      dplyr::group_by(...,  add = add)
  } 
}
```

Reply:
Resolved by https://github.com/r-lib/rlang/blob/master/R/compat-lazyeval.R